src/coreclr/vm/arm64/thunktemplates.S

```diff
@@ -120,9 +120,12 @@ LEAF_END_MARKED CallCountingStubCodeTemplate, _TEXT
     .irp STUB_PAGE_SIZE, 16384, 32768, 65536
```

Comment:
Does `FEATURE_MAP_THUNKS_FROM_IMAGE` above need the same change?

Reply:
Sigh, I missed that part of the merge. Yes.